### PR TITLE
Merge 1.1.0 in master

### PR DESCRIPTION
**Pull request by q-litzler**

### CODE REVIEW

File: docs/Places.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+Reverse geocoding means converting a location (latitude and longitude) to a readable address.
+
+```kotlin
+client.reverseGeocoding(Point(40.7128f, -74.0060f)) // New-York
+```
+
+#### Get By ObjectID
+
+Use a Places `objectID` To get an Algolia Places record.
```
Comment (PLNech) [inline docs/Places.md]:
```suggestion
Use a Places `objectID` to get an Algolia Places record.
```

File: src/commonMain/kotlin/com/algolia/search/endpoint/EndpointPlaces.kt [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+public interface EndpointPlaces {
+
+    suspend fun searchPlaces(
+        query: PlacesQuery = PlacesQuery(),
+        requestOptions: RequestOptions? = null
+    ): ResponseSearchPlacesMulti
+
+    suspend fun searchPlaces(
+        language: Language,
```
Comment (PLNech) [inline src/commonMain/kotlin/com/algolia/search/endpoint/EndpointPlaces.kt]: Why overload instead of single method with `language: Language? = null`?

Comment (q-litzler) [reply src/commonMain/kotlin/com/algolia/search/endpoint/EndpointPlaces.kt]: The Places search endpoint has a quirk: If you pass a `language` parameter, the JSON response will have a completely different structure.
So I need two separate methods to be able to parse the two different responses accordingly.

Comment (PLNech) [reply src/commonMain/kotlin/com/algolia/search/endpoint/EndpointPlaces.kt]: Ah right :confused:

**Pull request merged**